src/styles/contacts.styl

```diff
@@ -6,6 +6,10 @@ main > [role=main]
     max-width 80rem
     margin 0 auto
 
+[role=banner] .coz-bar-container
+    +small-screen()
+        align-items center
+
```

Comment:
Can you open an issue on the cozy-bar repo for this issue ? It seems like align-items center should be the default for `coz-bar-container` on mobile.

Reply:
issue : https://github.com/cozy/cozy-bar/issues/720